stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -754,6 +781,7 @@ internal class ChannelLogic(
         if (changedMessages.isNotEmpty()) {
             upsertMessages(changedMessages)
         }
+        mutableState._loading
```

Comment:
this is useless, right?

Reply:
yes, must have left it by accident after testing